src/workloads/execution/SecondaryReadsGenny.yml

```diff
@@ -0,0 +1,231 @@
+SchemaVersion: 2018-07-01
+Owner: "@TODO"  # what should go here? us or the owners of secondary_reads?
+Description: >
+ Test performance of secondary reads while applying oplogs
+
+ ### Setup
+
+ This workload runs only in a replica set with EXACTLY 3 NODES.
+ To use it in a replica set with a different number of nodes, adapt the
+ *number_of_nodes parameter (needs to be >1).
+
+ ### Test
+
+ On an empty replica set, the workload performs the following actions:
+ - Setup stage:
+    - Inserted *initial_nb_docs* documents into both the primary and the
+      secondaries. This should be done before any writes and reads.
+ - Actual test stage:
+    - *nWriters* background writers doing infinite writes on the primary which
+      will replicate these writes to secondaries.
+    - 1/16/32 readers doing queries on a sepcified seconary. Each query has
+      *batch_size* numbers of documents and only reads documents inserted in the
+      setup stage in order to have more constant behavior all the time.
+    - Finally, a query with WriteConcern 3 is used to measure the replication lag
+ - Cleanup stage:
+   - Drop the collection to reset the db SortPattern:
+ The test cycles through these stages for 1, 16, and 32 reader threads.
+
+
+Initial_nb_docs: &initial_nb_docs 10000
+DocumentSize: &document_size 459  # Target document size of 500 and Object.bsonsize({_id: ObjectId(), x: 0, str: "x"}) + 1 = 41
+ReadLimit: &read_limit 100
+MaxIdForFinds: &max_id_for_finds 9900
+TestDurationInSecs: &phase_duration 180 seconds
+Db: &db test
+WriterThreads: &writer_threads 16
+NumPhases: &max_phase 13
+NumberOfNodes: &number_of_nodes 3
+
+# This client allows the ReadsOnSecondary actor to directrly connect to the secondary node.
```

Comment:
[nit] "directly"

Reply:
Fixed, maybe it's time to install a spellchecker in my VSCode :)